aea/aea.py

```diff
@@ -71,14 +71,16 @@ def __init__(
         :return: None
         """
         super().__init__(
-            name=name,
+            identity=identity,
             connections=connections,
             loop=loop,
             timeout=timeout,
             debug=debug,
             programmatic=programmatic,
         )
 
+        self.identity.public_keys = wallet.public_keys
+        self.identity.addresses = wallet.addresses
         self._wallet = wallet
```

Comment:
the wallet is now only used by the decision maker, as intended. we could therefore pass it to the decision maker and not keep it in the AEA.

Reply:
@marcofavorito keep it in AEA or move to DM? I tend to think we should do the latter

Reply:
ok do the latter